metadata.patch

```diff
@@ -0,0 +1,97 @@
+   </description>
+-  <url type="homepage">https://github.com/Inochi2D/inochi-creator</url>
+-  <url type="help">https://github.com/Inochi2D/inochi-creator</url>
+-  <url type="bugtracker">https://github.com/Inochi2D/inochi-creator/issues</url>
+   <screenshots>
+     <screenshot type="default">
+       <caption>Title screen</caption>
+-      <image>https://user-images.githubusercontent.com/7032834/170948484-2f3a8175-9c45-4cf6-ae7e-0a6ee86bdc53.png</image>
++      <image type="source">https://user-images.githubusercontent.com/7032834/170948484-2f3a8175-9c45-4cf6-ae7e-0a6ee86bdc53.png</image>
+     </screenshot>
+   </screenshots>
++  <releases>
++    <release version="v0.7.3" date="2022-07-03" type="stable">
++      <url>https://github.com/Inochi2D/inochi-creator/releases/tag/v0.7.3</url>
++    </release>
++    <release version="v0.7.2" date="2022-04-23" type="stable">
++      <url>https://github.com/Inochi2D/inochi-creator/releases/tag/v0.7.2</url>
++    </release>
++    <release version="v0.7.1" date="2022-03-29" type="stable">
++      <url>https://github.com/Inochi2D/inochi-creator/releases/tag/v0.7.1</url>
++    </release>
++    <release version="v0.7.0" date="2022-03-27" type="stable">
++      <url>https://github.com/Inochi2D/inochi-creator/releases/tag/v0.7.0</url>
++    </release>
++    <release version="v0.6.3" date="2022-02-23" type="stable">
++      <url>https://github.com/Inochi2D/inochi-creator/releases/tag/v0.6.3</url>
++    </release>
++    <release version="v0.6.2" date="2022-01-25" type="stable">
++      <url>https://github.com/Inochi2D/inochi-creator/releases/tag/v0.6.2</url>
++    </release>
++    <release version="v0.6.1" date="2022-01-25" type="stable">
++      <url>https://github.com/Inochi2D/inochi-creator/releases/tag/v0.6.1</url>
++    </release>
++    <release version="v0.6.0" date="2022-01-24" type="stable">
++      <url>https://github.com/Inochi2D/inochi-creator/releases/tag/v0.6.0</url>
++    </release>
++    <release version="v0.5.6-beta.1" date="2021-09-25" type="development">
++      <url>https://github.com/Inochi2D/inochi-creator/releases/tag/v0.5.6-beta.1</url>
++    </release>
++    <release version="v0.5.5-beta.1" date="2021-09-15" type="development">
++      <url>https://github.com/Inochi2D/inochi-creator/releases/tag/v0.5.5-beta.1</url>
++    </release>
++    <release version="v0.5.0-beta.3" date="2021-06-22" type="stable">
++      <url>https://github.com/Inochi2D/inochi-creator/releases/tag/v0.5.0-beta.3</url>
++    </release>
++    <release version="v0.5.0-beta.2" date="2021-06-19" type="development">
++      <url>https://github.com/Inochi2D/inochi-creator/releases/tag/v0.5.0-beta.2</url>
++    </release>
++    <release version="v0.5.0-beta.1" date="2021-06-06" type="stable">
++      <url>https://github.com/Inochi2D/inochi-creator/releases/tag/v0.5.0-beta.1</url>
++    </release>
++  </releases>
++  <url type="homepage">https://github.com/Inochi2D/inochi-creator</url>
```

Comment:
why no using the actual website that match the app-id

Reply:
https://inochi2d.com/inochi-creator isn't yet a valid URL, but I made a comment about it upstream, so hopefully it gets added. Although, maybe `https://inochi2d.com/#download` as a URL would work too?

Reply:
Hm, on second thought, maybe the itch.io link would work as well?

Reply:
I went with itch.io, as it's a better fit.

Reply:
> https://inochi2d.com/inochi-creator isn't yet a valid URL, 

Just the domain matter. It's always better than the github.